migrations/201803202020.do.3423-multiple-db-support.sql

```diff
@@ -0,0 +1,3 @@
+delete from couchdb_progress where seq = '0';
+alter table couchdb_progress add column source_id varchar not null;
+CREATE UNIQUE INDEX couchdb_progress_source_id ON couchdb_progress ( source_id );
```

Comment:
> delete from couchdb_progress where seq = '0';

Could you help me understand the intention behind this line? Is it supposed to clear out the existing progress value so that:

> alter table couchdb_progress add column source_id varchar not null;

Doesn't error when it encounters existing rows?

Regardless of that, we **definitely** don't want to lose the current seq we're up to (we have some massive databases, having to sync everything again would be really undesired), so this migration has to be a bit smarter unfortunately.

I also have no idea if anyone did this, but people external to medic have expressed interest in using couch2pg, and so there may actually be people using this in the wild, and we definitely don't want to mess with their installations either!